docs/user-guide/api-mediation/configuration-multi-tenancy-routing.md

```diff
@@ -0,0 +1,29 @@
+# Configuring routing in a multi-tenant environment
+
+In a multi-sysplex environment, the API Gateway may also need to register with a Central Discovery Service which gathers information about all installed API Gateways
```

Comment:
Since we're saying what else the API Gateway may need to register with to register with a central Discovery service, it would be good to state what the Gateway typically registers with when not in a multi-sysplex environment.  Something like, "In addition to the API Gateway registration with .... required for non multi-sysplex environments, in a muti-sysplex environment, the API Gateway may need to register with a Central Discovery Service...."